man/busctl.xml

```diff
@@ -78,10 +78,10 @@
         <term><command>capture</command> <arg choice="opt" rep="repeat"><replaceable>SERVICE</replaceable></arg></term>
 
         <listitem><para>Similar to <command>monitor</command> but
-        writes the output in pcap format (for details, see the <ulink
-        url="https://wiki.wireshark.org/Development/LibpcapFileFormat">Libpcap
-        File Format</ulink> description). Make sure to redirect
-        standard output to a file. Tools like
+        writes the output in pcapng format (for details, see
+        <ulink url="https://github.com/pcapng/pcapng/">
+        PCAP Next Generation (pcapng) Capture File Format</ulink>).
+        Make sure to redirect standard output to a file or pipe. Tools like
```

Comment:
what happens with no redirection? is it a text output or will it print garbage to the screen?

Reply:
Existing code has a check for isatty() and refuses to allow binary data to go there.
Not what I would have done, but that was what original author did.

Reply:
try it
```
$ busctl capture
Refusing to write message data to console, please redirect output to a file.

Reply:
I think that's fine.